configs/opencv-ovms/demos/classification/python/classification_demo.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python3
+"""
+ Copyright (C) 2018-2023 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import logging as log
+import sys
+from argparse import ArgumentParser, SUPPRESS
+from pathlib import Path
+from time import perf_counter
+
+import os
+import paho.mqtt.client as mqtt
+import json
+
+import cv2
+
+sys.path.append(str(Path(__file__).resolve().parents[2] / 'common/python'))
```

Comment:
would it be useful to try and install these packages as part of setup instead? we can build/create a localized python wheel that can then be installed using pip...

Reply:
we can add the improvement in the future